import/faker/src/faker.utils.ts

```
@@ -0,0 +1,185 @@
+  return faker.number.int()
+}
+
+function generateDateData(label: string): string {
+  if (label.toLowerCase().includes('birth')) {
+    return faker.date.past({ years: 80 }).toDateString()
+  }
+  return faker.date.recent({ days: 1 }).toDateString()
+}
+
+function generateEnumData(config: any): string {
+  if (config && config.options) {
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)